### ISSUE TITLE
add build directory to gitignore

### ISSUE THREAD

**Issue by realstealthninja** [CaedenPH/portfolio#1]

**Comment by CaedenPH** [comment]
@realstealthninja I use the `build` directory in order to host the website from my aws ec2 instance. It is easier to push the `build` to github, because the instance lacks the ram requires to build the project
If you have any better methods let me know

**Comment by realstealthninja** [comment]
from a bit of research i think it might be possible to build using github actions. 
@CaedenPH

**Comment by CaedenPH** [comment]
> from a bit of research i think it might be possible to build using github actions. @CaedenPH

Yes, I could build it using github actions but it didn't seem towork as well as I wanted to when I implemented it. You can make a pr implemented this if you think it is the ideal way to build this project

**Comment by realstealthninja** [comment]
how would i be able to test if it builds? 
https://docs.github.com/en/actions/deployment/deploying-to-your-cloud-provider/deploying-to-amazon-elastic-container-service

this seems promising

**Comment by CaedenPH** [comment]
> 

Not sure how you would test it
